app/views/admin/shared/_nested_sidebar.html.erb

```diff
@@ -40,3 +40,10 @@
     <% end %>
   </li>
 <% end %>
+
+<script>
+  $('.js-menu-activator').on('show.bs.collapse', function (event) {
```

Comment:
Some notes on this:
1. I was hoping that I could have this bootstrap event handler code in the Sidebar Stimulus controller. I tried using the initialize() and even the connect() function, as well as using onLoad but the event handler never gets called from within any of them.
2. I thought that I could use Javascript here instead of JQuery, sort of like:
 ```
 document.querySelector(<whatever the element is>).addEventListener("show.bs.collapse", function() {
  });
```
but again this doesn't get called and then I came across this [stackoverflow that you can't use jQuery custom events with the regular JavaScript event engine](https://stackoverflow.com/questions/34089457/are-jquery-events-usables-with-dom-addeventlistener).

Reply:
Ah ah the gift that keeps on giving :D

Reply:
Tell me about it 😅